### ISSUE TITLE
Regression: Creating new classes from the CLI fails

### ISSUE THREAD

**Issue by sandermarechal** [phpactor/phpactor#792]
It seems that #496 and #489 broke generating new classes from the CLI. For example:

```
$ phpactor cl:new --variant=controller src/My/NewClass.php
src:src/My/NewClass.php
path:<?php

namespace My;

class NewClass
{
}
exists:false
```
Two problems:
1. The variant isn't used. Although the `--list` option does show my variant correctly.
2. The source code is output in the `path` field

**Comment by sandermarechal** [comment]
Reverting to 0.11.1 fixes the variant issue, but still prints the class source code instead of the file path. I need to revert all the way back to 0.4.0 to fix the path display issue.

**Comment by dantleech** [comment]
can confirm the path/src issue. On both `develop` and `master` I cannot reproduce the variant issue.

**Comment by dantleech** [comment]
I've added more tests to cover variants (they seem to work) and fixed the path/src confusion here: https://github.com/phpactor/phpactor/pull/793

**Comment by sandermarechal** [comment]
I can confirm that everything now works as expected on develop. The variant issue only happens on 0.12.0/master, not on develop.

**Comment by dantleech** [comment]
ok, it should be fixed in develop as of #766 (which is merged in develop but not yet master), closing for now.
